packages/block-editor/src/components/link-control/index.js

```diff
@@ -534,7 +540,9 @@ function LinkControl( {
 							stopEditing();
 						}
 					} }
-					renderSuggestions={ renderSearchResults }
+					renderSuggestions={
+						showSuggestions ? renderSearchResults : false
```

Comment:
As `renderSuggestions` is returning React elements would it be better to return `null` here to indicate that we don't wish to render anything.

https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
Yes, good catch :)